app/src/main/res/values/styles.xml

```diff
@@ -537,6 +537,7 @@
         <item name="elevationOverlayEnabled">false</item>
         <item name="materialAlertDialogTitleTextStyle">@style/H2.DialogTitle</item>
         <item name="materialAlertDialogBodyTextStyle">@style/P.DialogBody</item>
+        <item name="android:windowSoftInputMode">adjustResize</item>
```

Comment:
It looks like the "How can we improve the feature" in the Edit Patrol is not handled properly after applying it.

Would it be possible to define it in an individual style, for example: `AlertDialogTheme.Resize` or another better name so that we can use it only when we need it?

Reply:
Hmm ok then, let's create a child style, but also promise to revisit and consolidate these dialogs to use a single style in the future.